app/src/main/java/com/kylecorry/trail_sense/navigation/ui/NavigatorFragment.kt

```diff
@@ -649,6 +651,23 @@ class NavigatorFragment : BoundFragment<ActivityNavigatorBinding>() {
         binding.location.text = formatService.formatLocation(gps.location)
 
         updateNavigationButton()
+
+        // show on lock screen
+        if (userPrefs.navigation.lockScreenPresence) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O_MR1) {
```

Comment:
Can the lock screen presence be turned off when you navigate away from the navigation tab? (i.e. it should only be active when you navigate to a beacon and lock your screen while on the navigation tab)

Reply:
Fixed in https://github.com/kylecorry31/Trail-Sense/pull/677/commits/b5322cdef7fe649f81568cad582097802f26a4ca